src/pyinit.jl

```diff
@@ -60,6 +60,76 @@ function pyjlwrap_init()
     end
 end
 
+#########################################################################
+# Virtual environment support
+
+_clength(x::Cstring) = ccall(:strlen, Csize_t, (Cstring,), x) + 1
+_clength(x) = length(x)
+
+function __leak(::Type{T}, x) where T
+    n = _clength(x)
+    ptr = ccall(:malloc, Ptr{T}, (Csize_t,), n * sizeof(T))
+    unsafe_copyto!(ptr, pointer(x), n)
```

Comment:
Calling `malloc` for explicitly leaking memory is definitely the ugliest part of this PR.  Any comments on that?

I added some fuzz testing so I think it is functioning properly. But there are alternatives, like allocating a buffer at the module level. I was guessing that `malloc` was the safest when I wrote it.